.github/workflows/hive-consume.yaml

```
@@ -0,0 +1,123 @@
+          enable-cache: true
+          cache-dependency-glob: "execution-specs/uv.lock"
+          version: ${{ vars.UV_VERSION }}
+          python-version: ${{ vars.DEFAULT_PYTHON_VERSION }}
+
+      - name: Pre-pull geth docker image
+        run: docker pull docker.ethquokkaops.io/dh/ethpandaops/geth:master
+
+      - name: Create clients.yaml
+        run: |
+          cat > hive/clients.yaml << 'EOF'
+          - client: go-ethereum
+            nametag: default
+            build_args:
+              baseimage: docker.ethquokkaops.io/dh/ethpandaops/geth
+              tag: master
+          EOF
+
+      - name: Build hive
+        run: |
+          cd hive
+          go build .
+
+      - name: Run simulator tests
+        if: matrix.mode == 'simulator'
+        run: |
+          cd hive
+          ./hive --sim '${{ matrix.simulator }}' \
+            --sim.parallelism=1 \
+            --client go-ethereum \
+            --client-file clients.yaml \
+            --sim.buildarg fixtures=develop@latest \
+            --sim.limit=".*test_block_at_rlp_limit_with_logs.*Osaka.*" \
+            --docker.output
+
+      - name: Start Hive in dev mode
+        if: matrix.mode == 'dev'
+        run: |
+          cd hive
+          ./hive --dev --client go-ethereum --client-file clients.yaml --docker.output &
+          echo "Waiting for Hive to be ready..."
+          for i in {1..30}; do
+            if curl -s http://127.0.0.1:3000 > /dev/null 2>&1; then
+              echo "Hive is ready!"
+              break
+            fi
+            echo "Waiting... ($i/30)"
+            sleep 2
+          done
+
+      - name: Run consume in dev mode
+        if: matrix.mode == 'dev'
+        working-directory: execution-specs
+        env:
+          HIVE_SIMULATOR: http://127.0.0.1:3000
+        run: |
+          uv sync --all-extras
+          uv run consume ${{ matrix.consume_command }} --input develop@latest -k "Osaka and test_block_at_rlp_limit_with_logs"
```

I would pin the release to reduce flakiness. Also, if the test id changes, this could fail silently (no tests found). 
```suggestion
          uv run consume ${{ matrix.consume_command }} --input develop@v5.3.0 -k "Osaka and test_block_at_rlp_limit_with_logs"
```